src/officehours_ui/tests.py

```diff
@@ -0,0 +1,12 @@
+from django.test import TestCase, Client
+
+class UITestCase(TestCase):
+    def test_robots_txt(self):
+        c = Client()
+        response = c.get('/robots.txt')
+        self.assertEqual(response.status_code, 200)
+
+    def test_favicon_ico(self):
+        c = Client()
+        response = c.get('/static/favicon.ico')
+        self.assertEqual(response.status_code, 200)
```

Comment:
I think these tests need to be modified. The first one is failing because of the redirect (getting status code 301). The other works, but it's not testing the root, which is I think where we typically see requests to. I guess both should be change to hit the root plus the filename, and they should expect 301?

Reply:
Okay I didn't know about the tests, I made the updates to have them both get a 301. It looks like this is the correct return.